R/calcCCScapacity.R

```diff
@@ -9,7 +9,9 @@
 #'
 #' @export
 calcCCScapacity <- function(subtype) {
-  x <- calcOutput("ProjectPipelines", subtype = "CCS", aggregate = F)
+
+  x <- calcOutput("ProjectPipelines", subtype = "CCS",
+                  aggregate = FALSE, warnNA = FALSE)
```

Comment:
are we sure, that NAs are o.k. for this function?

Reply:
Yes, from my pov thats ok.